common/src/main/scala/explore/components/Tile.scala

```diff
@@ -75,8 +73,8 @@ object Tile {
   val component =
     ScalaFnComponent
       .withHooks[Props]
-      // info ref
-      .useRefToVdom[html.Element]
+      // infoRef - We use this mechanism instead of a regular Ref in order to force a rerender when it's set.
+      .useStateView(none[html.Element])
```

Comment:
Would you recommend in genera not to use `useRefXXX`?

Reply:
`useRef*` has its uses, it's just that it doesn't trigger a rerender like `useState*` does.